internal/scaffold/ansible/molecule_test_local_playbook.go

```diff
@@ -44,115 +44,34 @@ const moleculeTestLocalPlaybookAnsibleTmpl = `---
   tasks:
-  - block:
-    - name: Delete the Operator Deployment
-      k8s:
-        state: absent
-        namespace: '{{ namespace }}'
-        definition: "{{ lookup('template', '/'.join([deploy_dir, 'operator.yaml'])) }}"
-      register: delete_deployment
-      when: hostvars[groups.k8s.0].build_cmd.changed
-
-    - name: Wait 30s for Operator Deployment to terminate
-      k8s_info:
-        api_version: '{{ definition.apiVersion }}'
-        kind: '{{ definition.kind }}'
-        namespace: '{{ namespace }}'
-        name: '{{ definition.metadata.name }}'
-      vars:
-        definition: "{{ lookup('template', '/'.join([deploy_dir, 'operator.yaml'])) | from_yaml }}"
-      register: deployment
-      until: not deployment.resources
-      delay: 3
-      retries: 10
-      when: delete_deployment.changed
-
     - name: Create the Operator Deployment
       k8s:
         namespace: '{{ namespace }}'
-        definition: "{{ lookup('template', '/'.join([deploy_dir, 'operator.yaml'])) }}"
-
-    - name: Create the [[.Resource.FullGroup]]/[[.Resource.Version]].[[.Resource.Kind]]
-      k8s:
-        state: present
-        namespace: '{{ namespace }}'
-        definition: '{{ custom_resource }}'
-
-    - name: Wait 2m for reconciliation to run
-      k8s_info:
-        api_version: '{{ custom_resource.apiVersion }}'
-        kind: '{{ custom_resource.kind }}'
-        namespace: '{{ namespace }}'
-        name: '{{ custom_resource.metadata.name }}'
-      register: cr
-      until:
-      - "'Successful' in (cr | json_query('resources[].status.conditions[].reason'))"
-      delay: 12
-      retries: 10
-    rescue:
-    - name: debug cr
-      ignore_errors: yes
-      failed_when: false
-      debug:
-        var: debug_cr
-      vars:
-        debug_cr: '{{ lookup("k8s",
-          kind=custom_resource.kind,
-          api_version=custom_resource.apiVersion,
-          namespace=namespace,
-          resource_name=custom_resource.metadata.name
-        )}}'
-
-    - name: debug memcached lookup
-      ignore_errors: yes
-      failed_when: false
-      debug:
-        var: deploy
+        definition: "{{ lookup('template', operator_template) }}"
+        wait: yes
       vars:
-        deploy: '{{ lookup("k8s",
-          kind="Deployment",
-          api_version="apps/v1",
-          namespace=namespace,
-          label_selector="app=memcached"
-        )}}'
-
-    - name: get operator logs
-      ignore_errors: yes
-      failed_when: false
-      command: kubectl logs deployment/{{ definition.metadata.name }} -n {{ namespace }}
-      environment:
-        KUBECONFIG: '{{ lookup("env", "KUBECONFIG") }}'
-      vars:
-        definition: "{{ lookup('template', '/'.join([deploy_dir, 'operator.yaml'])) | from_yaml }}"
-      register: log
-
-    - debug: var=log.stdout_lines
-
-    - fail:
-        msg: "Failed on action: converge"
-
-- import_playbook: '{{ playbook_dir }}/../default/asserts.yml'
+        pull_policy: Never
```

Comment:
Same note about the `pull_policy` var being removed and hardcoded in the template; either remove this var or make that override-able again.